src/contexts/tour-context/context.tsx

```diff
@@ -0,0 +1,876 @@
+                        HSS provides a histogram filter to zoom in on higher-scoring results (those more
+                        relevant to the search query). Hovering the mouse pointer over each variable in the
+                        histogram displays additional information, including its name, description, and study.
+                    </div>
+                ),
+                buttons: [
+                    {
+                        classes: secondaryButtonClass,
+                        text: 'Back',
+                        action: function() {
+                            const conceptGridViewOption = getConceptViewRadioOption()!
+                            this.back()
+                            conceptGridViewOption.click()
+                        }
+                    },
+                    {
+                        classes: primaryButtonClass,
+                        text: 'Next',
+                        type: 'next'
+                    }
+                ],
+            },
+            {
+                id: "main.search.variable-view.study-list",
+                path: searchPathPattern,
+                attachTo: {
+                    element: studyListItemDomMask.selector,
+                    on: "bottom"
+                },
+                scrollToHandler: () => scrollIntoViewIfNeeded(studyListItemDomMask.originalSelector),
+                text: renderToStaticMarkup(
+                    <div>
+                        You can also click to expand each study to view the study variables associated
+                        with the search query.
+                    </div>
+                ),
+                buttons: [
+                    {
+                        classes: secondaryButtonClass,
+                        text: 'Back',
+                        type: 'back',
+                    },
+                    {
+                        classes: primaryButtonClass,
+                        text: 'Next',
+                        type: 'next'
+                    }
+                ],
+                when: {
+                    show: () => { studyListItemDomMask.showMask() },
+                    hide: () => { studyListItemDomMask.hideMask() },
+                    cancel: () => { studyListItemDomMask.hideMask() },
+                    complete: () => { studyListItemDomMask.hideMask() }
+                }
+            },
+            {
+                id: "main.outro",
+                path: searchPathPattern,
+                attachTo: {
+                    element: supportNavDomMask.selector,
+                    on: "bottom"
+                },
+                scrollToHandler: () => scrollIntoViewIfNeeded(supportNavDomMask.originalSelector),
+                title: "Conclusion",
+                text: renderToStaticMarkup(
+                    <div>
+                        This concludes the introductory tour of HSS. For more detailed information,
```

Comment:
```suggestion
                        This concludes the introductory tour of HSS. For more detailed information, click on the "Get Help" button to
```

Reply:
I don't really think it makes sense to tell them to "click on the get help button" when the "User Guide" text itself is a link and immediately after, it explains they can navigate to the support page.

Reply:
I think the wording should be changed from support page to get help now that this has been renamed, though.